### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by CyberDem0n**
We want to avoid ping-ping as much as possible.

### CODE REVIEW

**Comment by avaczi** [comment]
:+1:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```
Comment (sdudoladov) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: @CyberDem0n, is that field really necessary ?  you do 99% of work anyway

**Review by sdudoladov — APPROVED** [review]

**Comment by sdudoladov** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**